JavaAssignment/src/prachiJ/FahrenheitToCelsius.java

```diff
@@ -0,0 +1,18 @@
+package prachiJ;
+
+import prachiJ.FahrenheitToCelsius;
+
+public class FahrenheitToCelsius {
+
+    static void convertFToC(int f)
```

Comment:
Logic looks good but needs to work on adding comments, method name, and formatting before committing.